### ISSUE TITLE
Geo Distance Sorting

### ISSUE THREAD

**Issue by ManjeetMP** [danpaz/bodybuilder#58]
Attempting to do something along the lines of this:
https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-sort.html#geo-sorting

Any way to do geo distance sorting?

**Comment by danpaz** [comment]
The closest I think can be done right now is this:

```js
new Bodybuilder()
        .sort('_geo_distance', 'asc')
		.query('term', 'user', 'kimchy')
        .build()
/*
{
  "sort": [
    {
      "_geo_distance": {
        "order": "asc"
      }
    }
  ],
  "query": {
    "term": {
      "user": "kimchy"
    }
  }
}
*/
```

because it's not possible currently to append arbitrary options in `sort`. Tbh I didn't know this usage of sort existed so it could make a good enhancement to support this.

**Comment by danpaz** [comment]
Should be resolved by #107, feel free to reopen if its not working for you.
